admin/view-translations-term.php

```diff
@@ -27,11 +27,16 @@
 ?>
 <table class="widefat term-translations"  id="<?php echo isset( $term_id ) ? 'edit' : 'add'; ?>-term-translations">
 	<?php
+	$authorized_languages = $this->model->languages->filter( 'translator' )->get_list();
+	$authorized_slugs     = wp_list_pluck( $authorized_languages, 'slug' );
```

Comment:
Since `$authorized_languages` is not used:

```suggestion
	$authorized_slugs = $this->model->languages->filter( 'translator' )->get_list( array( 'fields' => 'slug' ) );
```